docs/Using Any API/large-responses.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: nodes.liquid
+section: smartContract
+date: Last Modified
+title: "Large Responses"
+permalink: "docs/large-responses/"
+whatsnext: {"Make an Existing Job Request":"/docs/existing-job-request/", "API Reference":"/docs/chainlink-framework/", "Contract Addresses":"/docs/decentralized-oracles-ethereum-mainnet/"}
+metadata: 
+  title: "Large Responses"
+  description: "Learn how to make a GET request to an API and receive a dynamically sized large response from a smart contract, using Chainlink.."
+  image: 
+    0: "/files/930cbb7-link.png"
+---
+This page explains how to make an HTTP GET request to an external API from a smart contract, using Chainlink's [Request & Receive Data](../request-and-receive-data/) cycle and then receive large responses.
+
+# Large Response
+
+To consume an API with a large responses, your contract should inherit from <a href="https://github.com/smartcontractkit/chainlink/blob/master/contracts/src/v0.8/ChainlinkClient.sol" target="_blank">`ChainlinkClient`</a>. This contract exposes a struct called `Chainlink.Request`, which your contract should use to build the API request. The request should include the oracle address, the job id, the fee, adapter parameters, and the callback function signature.
+
+>❗️ Remember to fund your contract with LINK!
+>
+> Making a GET request will fail unless your deployed contract has enough LINK to pay for it. **Learn how to [Acquire testnet LINK](../acquire-link/) and [Fund your contract](../fund-your-contract/)**.
+> 
+
+<div class="remix-callout">
+    <a href="https://remix.ethereum.org/#optimize=false&evmVersion=null&gist=10ae2499f48f2932b38f34a96b124443&runs=200&version=soljson-v0.8.6+commit.11564f7e.js" target="_blank" class="cl-button--ghost solidity-tracked">Deploy this contract using Remix ↗</a>
+    <a href="../deploy-your-first-contract/" title="">What is Remix?</a>
+</div>
+
+```javascript Kovan
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@chainlink/contracts/src/v0.8/ChainlinkClient.sol";
+
+/**
+ * @notice DO NOT USE THIS CODE IN PRODUCTION. This is an example contract. 
+ */
+contract GenericBigWord is ChainlinkClient {
+  using Chainlink for Chainlink.Request;
+
+  // variable bytes returned in a signle oracle response
+  bytes public data;
+  string public image_url;
+
+  /**
+   * @notice Initialize the link token and target oracle
+   * @dev The oracle address must be an Operator contract for multiword response
+   *
+   *
+   * Kovan Testnet details: 
+   * Link Token: 0xa36085F69e2889c224210F603D836748e7dC0088
+   * Oracle: 0xc57B33452b4F7BB189bB5AfaE9cc4aBa1f7a4FD8 (Chainlink DevRel)
+   *
+   */
+  constructor(
+  ) {
+    setChainlinkToken(0xa36085F69e2889c224210F603D836748e7dC0088);
+    setChainlinkOracle(0xc57B33452b4F7BB189bB5AfaE9cc4aBa1f7a4FD8);
+  }
+
+  ////---- VARIABLE BYTES ARRAY RETURNED BY ORACLE ----////
```

Comment:
We mostly use the following comments style for section or function descriptions:
```
    /**
    * Comment text
    */
```

For smaller comments, I've seen `// comment text` variation but I haven't seen `////----` one.
Should we replace it?